### ISSUE TITLE
Add new fields relating to string file info items

### ISSUE THREAD

**Issue by delphidabbler** [delphidabbler/vied#49]
This issue has been created from off topic suggestions in issue #31. Those comments were:

----

A field that returns the full contents of the _Original File Name_ string information item could be useful.

There's already the `<SHORTFNAME>` field that removes the file extension from _Original File Name_, so how about  `<FNAME>` for the full content.

_Originally posted by @delphidabbler in https://github.com/delphidabbler/vied/issues/31#issuecomment-1228941937_

----

Another useful field that provides access to a string information item could be `<COMPANYNAME>`. This would allow the company name to be used in string information, such as copyright.

Also, an obvious complement to `<SPECIALBUILD>` would be `<PRIVATEBUILD>`.

Finally, in case the _Internal Name_ string information isn't connected with _Original File Name_ it will do no harm to have a `<INTERNALNAME>` field.

_Originally posted by @delphidabbler in https://github.com/delphidabbler/vied/issues/31#issuecomment-1229077404_

----

In proposing additional fields providing access to string information items, wouldn't it be easier to provide a field for each string item ? That way there's no need to try to second guess which ones would be useful.

_Originally posted by @delphidabbler in https://github.com/delphidabbler/vied/issues/31#issuecomment-1229081411_

**Comment by delphidabbler** [comment]
Decided to provide a field for each string file info item:

```
<COMMENTS>
<COMPANYNAME>
<FILEDESCRIPTION>
<FILEVERSION>
<INTERNALNAME>
<LEGALCOPYRIGHT>
<LEGALTRADEMARKS>
<ORIGINALFILENAME>
<PRIVATEBUILD>
<PRODUCTNAME>
<PRODUCTVERSION>
<SPECIALBUILD>
```

Of these, `<SPECIALBUILD>` and `<PRODUCTNAME>` already exist.

**Comment by delphidabbler** [comment]
> There's a bug waiting in the form of circular references between string information fields (in fact it might already exist between the current string info fields).

Confirmed this is a bug and reported as issue #58.

**Comment by delphidabbler** [comment]
Implemented at merge commit f44f3a6bd79dc694faf6367d572f6af430604998
